doc/operator-notes.md

```diff
@@ -103,3 +103,23 @@ Specifying `size` as 0 means the partition should span to the end of the largest
 ### Unspecified partition size
 If `size` is not specified and a partition with the same number exists, it will use the value of the existing partition, unless wipePartitionEntry is set.
 If `size` is not specified and there is no existing partition, or wipePartitionEntry is set, `size` act as if it were set to 0 and use the size of the largest block.
+
+## Config Merging
+
+Ignition supports fetching, then merging multiple configs. This replaces the `append` functionality of the Ignition 2.x.0 specification. There are several rules that determine how configs get merged. When a child config is merged with a parent, generally the child config's values override the parent config's values.
+
+### Child configs take precedence when specified
+
+If a parent and child object are being merged, the fields in the child object take precedence over the fields in the parent stuct. If a field in the child object is not specified, the field from the parent is used instead.
+
+### Most lists are deduplicated
+
+All lists of objects have a field that uniquely idenitifies that object. If a child config contains an entry that conflicts with the parent config, those entries are merged. A few sections of the config are exempt from this behavior. See the [configuration specification] for a complete listing. Generally the only lists that are simply appended are those that specify arguments to commands like `mkfs` or `mdadm`.
+
+### Files, Directories, and Links are deduplicated across each other
+
+Since files, directories, and links all describe filesystem entries can can conflict, these lists are deduplicated across each other. This means a file in child config can replace a link in the parent, or a directory in a child config can replace a file in the parent.
+
+### Configs are merged in a depth first traversal
```

Comment:
How about just "Configs are merged in the order they appear"? Seems clearer than "depth first traversal".

Reply:
That's more ambiguous. Consider if config A merged B and C, but B merges D and E. The final config looks like:
```
Final =
M(
    M(
        A,
        M(
            M(B, D),
            E
        )
    )
C)
```
Basically "order they appear" isn't well defined.